Fantasy Freaks/WinStat.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fantasy_Freaks {
+    internal class WinStat {
+        public static WinStat instance;
+
+        public WinStat() {
+            instance = this;
```

Comment:
What is the purpose of using this? Since it's static is shouldn't make a difference

Reply:
I see you're storing data in the variables of the static class. If that's what's causing you to need "this", you should make another class that you can pass as an object into WinStat methods.

Reply:
This portion was put in so that the class is always active throughout the running of the application, this allows for portions of the code to be edited/used throughout the running of the application. If you have another way (like another class) we should hop on a call and work this through unless you can cover it.

Reply:
Got it, artificially extending the lifetime of static variables by passing around an instance will end up complicating things, I'll go over the solution I commented later with you, so you can leave it as it is for now.